### PR TITLE
Update readme and add list animation example

### LINKED ISSUE #6: Bullet animation information
Beamer has a feature for animating billeted lists in pdf presentations. This issue can be closed when that feature is documented or indicated in the repo's README or in an example slide.

### PR DESCRIPTION
**Pull request by nsryan2**
This PR adds information about using the template (almost exactly the same as the report template), and using the animation feature for bulleted lists and closes #6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,91 @@
-# Presentation Templates
+# Purpose of this Repository:
+This repository is intended as a template to standardize and ease the implimentation
```
Comment (smpark7) [inline README.md]:
```suggestion
This repository is intended as a template to standardize and ease the implementation 
```

File: README.md [hunk 2]
```diff
@@ -1,11 +1,91 @@
...
+
+An important note is that the citation is made with what the example called "call_tag." 
+The first line in the example bibliography.bib code contains this tag, and connects the 
+content to the Reference section that will be generated in the document.
+
+
+
+# ARFC Report Manual
+For ARFC specific guidelines for making a report, read the specifics at
+http://arfc.npre.illinois.edu/manual/guides/writing/presentation
```
Comment (smpark7) [inline README.md]: Do you mean to say "presentation" instead of "report" here?

File: conclusion.tex [hunk 3]
```diff
@@ -1,10 +1,16 @@
 \begin{frame}
   \frametitle{Conclusion}
-        We showed many things.
+        We showed many things. This slide is an example of how 
+        you can animate bulletted lists, for more information about
```
Comment (smpark7) [inline conclusion.tex]:
```suggestion
        you can animate bulleted lists, for more information about
```

**Review by smpark7 — CHANGES_REQUESTED** [review]
I only have minor comments. Otherwise it looks great and thanks for introducing these improvements!

File: README.md (hunk 2 above)
Comment (nsryan2) [reply README.md]:
```suggestion
# ARFC Report Manual
For ARFC specific guidelines for making a presentation, read the specifics at
http://arfc.npre.illinois.edu/manual/guides/writing/presentation
```

**Comment by nsryan2** [comment]
Thanks for the feedback, should be all good to go now!

**Review by smpark7 — APPROVED** [review]

**Pull request merged**